### PR TITLE
Update CMakeLists to use raja_add_executable where applicable

### PR DESCRIPTION
**Pull request by davidbeckingsale**

### CODE REVIEW

File: Kripke-v1.1/Kripke-v1.1-RAJA/Kripke/CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 include_directories(..)
 
-add_definitions(-DRAJA_ENABLE_NESTED)
+if (NOT RAJA_ENABLE_NESTED)
```
Comment (rhornung67) [inline Kripke-v1.1/Kripke-v1.1-RAJA/Kripke/CMakeLists.txt]: Do we want to do it this way? Or, if forallN is disabled should we not build Kripke?

File: Kripke-v1.1/Kripke-v1.1-baseline/Kripke.h [hunk 2]
```diff
@@ -61,7 +61,7 @@
 
 
 // Adopt RAJA's use of OPENMP
-#include<RAJA/RAJA.hxx>
+#include "RAJA/config.hxx"
```
Comment (rhornung67) [inline Kripke-v1.1/Kripke-v1.1-baseline/Kripke.h]: :+1:

File: LULESH-v1.0/LULESH-v1.0_RAJA-variants/CMakeLists.txt [hunk 3]
```diff
@@ -46,24 +46,19 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR})
...
-    luleshRAJA-serial.cxx)
+endif()
 
-  add_dependencies(lulesh-RAJA-serial.exe RAJA)
+raja_add_executable(
+  NAME lulesh-RAJA-parallel.exe
+  SOURCES luleshRAJA-parallel.cxx
+  DEPENDS_ON ${RT_LIBRARIES})
 
+if (NOT RAJA_ENABLE_CUDA)
```
Comment (rhornung67) [inline LULESH-v1.0/LULESH-v1.0_RAJA-variants/CMakeLists.txt]: See here, we don't build the LULESH variants that use CUDA if CUDA is not enabled (see my comment related to Kripke)

File: Kripke-v1.1/Kripke-v1.1-RAJA/Kripke/CMakeLists.txt (hunk 1 above)
Comment (davidbeckingsale) [inline Kripke-v1.1/Kripke-v1.1-RAJA/Kripke/CMakeLists.txt]: Could do. To be honest, I'm not even sure the `RAJA_ENABLE_NESTED` option is that useful anymore.

Comment (rhornung67) [inline Kripke-v1.1/Kripke-v1.1-RAJA/Kripke/CMakeLists.txt]: Agreed. I have no objection to removing it if you wish.

File: Kripke-v1.1/Kripke-v1.1-baseline/Kripke.h (hunk 2 above)
Comment (davidbeckingsale) [inline Kripke-v1.1/Kripke-v1.1-baseline/Kripke.h]: Yeah, including everything was breaking some stuff :(

**Pull request merged**